### ISSUE TITLE
Remove all `.unwrap()` and `.expect()`

### ISSUE THREAD

**Issue by maxammann** [maplibre/maplibre-rs#92]
## 🤔 Expected Behavior

Return a Result for failable operations.

## 😯 Current Behavior
We use sometimes unwrap or expecte because of lazyness.

**Comment by maxammann** [comment]
Hello @iakev I'm merging in a rather large PR in the next 1h :) So its better to start working after that.
Apart from that you should be fine.

Maybe it could make sense to leave some `.expect()` if they are very unlikely, or if you are unlikely to hit them after setting up the map.

**Comment by maxammann** [comment]
Or probably tomorrow..

**Comment by iakev** [comment]
Okay, no problem will start working on it tomorrow. In case I find any
difficulties, I presume I can contact you right?

**Comment by maxammann** [comment]
Yes, absolutely. You can also reach me on matrix.org Its @maxammann

**Comment by iakev** [comment]
Hello, I hope you are fine. So I have tweaked one file just wanted to hear
if what am doing is correct so that I can continue. Can I push the branch
for your review?

Best Regards,
Kevin Mwongera

**Comment by maxammann** [comment]
Sure, just push the branch to your fork and then open a PR in "Draft Mode": https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by iakev** [comment]
Thanks, I will do that. Kindly let me know your thoughts.

**Comment by xinslu** [comment]
Hi,
Is this issue still open? If yes, then I'd love to get involved!

**Comment by maxammann** [comment]
Hi! The description and goal of this issue needs some refinement. Ill try to refine it this weekend!
